rpi_backlight.py

```diff
@@ -144,9 +222,21 @@ def _create_argument_parser() -> argparse.ArgumentParser:
     )
     return parser
 
+def init() -> None:
+    global mode, path
+    if 'Raspberry Pi' in open('/sys/firmware/devicetree/base/model').read():
+        path = "/sys/class/backlight/rpi_backlight/"
+        mode = "MODE_RPI"
+    elif 'Tinker Board' in open('/sys/firmware/devicetree/base/model').read():
+        path = "/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/"
+        mode = "MODE_TINKERBOARD"
+    else:
+        print("Error: Could not detect OS.")
+        sys.exit()
```

Comment:
`Instead` of print and `sys.exit` let's throw an exception which can be handled by the code using this library.

Reply:
fixed